numpy_eigen/src/autogen_module/numpy_eigen_export_module.cpp

```diff
@@ -253,6 +253,8 @@ BOOST_PYTHON_MODULE(libnumpy_eigen)
 {
 	using namespace boost::python;
 	// Without this import, the converter will segfault
+#undef NUMPY_IMPORT_ARRAY_RETVAL
+#define NUMPY_IMPORT_ARRAY_RETVAL
```

Comment:
Can you keep the indentation via tabs so that it aligns with the rest of the code?

Reply:
We generally don't indent pre-processor macros. I'd accept hat.